### PR TITLE
Correct SKU endpoints

### PR DESCRIPTION
**Pull request by julia-rabello**
[Preview](https://preview.readme.io/?url=https://raw.githubusercontent.com/vtex/openapi-schemas/Correct-SKU-endpoints/VTEX%20-%20Catalog%20API.json)

#### Types of changes
- [x] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [x] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Catalog API.json [hunk 1]
```diff
@@ -4497,225 +4497,175 @@
...
-                            },
-                            "X-Track": {
-                                "content": {
-                                    "text/plain": {
-                                        "schema": {
-                                            "type": "string"
+                                        "WeightKg": {
+                                            "title": "WeightKg",
+                                            "type": "number",
+                                            "description": "SKU Weight."
```
Comment (karenkrieger) [inline VTEX - Catalog API.json:4603]: `weight`

File: VTEX - Catalog API.json [hunk 2]
```diff
@@ -4497,225 +4497,175 @@
...
+                                        },
+                                        "ModalType": {
+                                            "title": "ModalType",
+                                            "type": "string",
+                                            "description": "Links an unusual type of product to a carrier specialized in delivering it. To learn more about this feature, read our articles [How the modal works](https://help.vtex.com/en/tutorial/how-does-the-modal-work--tutorials_125) and [Setting up modal for carriers](https://help.vtex.com/en/tutorial/configure-modal--3jhLqxuPhuiq24UoykCcqy)."
+                                        },
+                                        "KitItensSellApart": {
+                                            "type": "boolean",
+                                            "title": "KitItensSellApart",
+                                            "description": "Defines if Kit components can be sold apart"
```
Comment (karenkrieger) [inline VTEX - Catalog API.json:4659]: `.`

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Catalog API.json (hunk 2 above)
Comment (julia-rabello) [reply VTEX - Catalog API.json:4659]: Já tinha sido resolvido

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**